packages/design-system-tokens/src/copy_themes.sh

```diff
@@ -13,16 +13,15 @@ copyThemes()
   
   for i in "${!PATHS[@]}"; do
     cp -v "${i}-layout-tokens.scss" "${PATHS[$i]}/_layout.scss"
-    cp -v "${i}-components-theme.css" "${PATHS[$i]}/theme/"
-    cp -v "${i}-theme.css" "${PATHS[$i]}/theme/"
-  done
-
-  #move files used in internal tooling
-  for file in *.scss; do
-    cp -v "$file" "../../docs/src/styles/theme-variables/_${file}"
-  done
-  for file in *.css; do
-    cp -v "$file" "../../docs/src/styles/theme-variables/_${file}"
+    rm "${PATHS[$i]}/${i}-theme.css"
+    echo ":root, ::before, ::after, ::backdrop {" >> "${PATHS[$i]}/${i}-theme.css"
+    cat "${i}-theme.css" >> "${PATHS[$i]}/${i}-theme.css"
+    cat "${i}-components-theme.css" >> "${PATHS[$i]}/${i}-theme.css"
+    echo "}" >> "${PATHS[$i]}/${i}-theme.css"
+    # copy to storybook static for storybook building
+    cp -v "${PATHS[$i]}/${i}-theme.css" ../../../.storybook/static
+    mkdir -p ../../docs/static/themes
+    cp -v "${PATHS[$i]}/${i}-theme.css" ../../docs/static/themes
```

Comment:
It might be easier to read if `"${PATHS[$i]}/${i}-theme.css"` were turned into a local variable. It looks like it's being used six times, so it could be justified.

Reply:
> I haven't dug into why, but the little arrow icons aren't showing for me on the dropdowns. I just stumbled upon that randomly. Might have to do with the image path variable.

Hmm that's odd, they're showing up for me.

![image](https://user-images.githubusercontent.com/783820/208765740-ab41d918-1ca4-4746-8a3a-bf4a0d60ea66.png)

Reply:
I cleaned and built again, and it wasn't a problem anymore. False alaram